### PR TITLE
Fixup failing fuchsia tests

### PR DESCRIPTION
**Pull request by c6c7**
The Fuchsia platform passes all tests with these changes. Two tests are ignored because they rely on Fuchsia not returning a status code upon a process aborting. See #102032 and #58590 for more details on that topic.

Many formatting changes are also included in this PR.

r? tmandry
r? erickt

### CODE REVIEW

**Comment by c6c7** [comment]
See https://ci.chromium.org/ui/p/fuchsia/builders/try/rust_test-x64-linux/b8743058030660555825/overview for verification that these changes fix the failing tests.

**Comment by c6c7** [comment]
The failures were due to adding a line to the source of two tests and the expected output containing line numbers.

File: tests/ui/test-attrs/test-panic-abort-nocapture.rs [hunk 1]
```diff
@@ -10,6 +10,7 @@
 //@ ignore-wasm no panic or subprocess support
 //@ ignore-emscripten no panic or subprocess support
 //@ ignore-sgx no subprocess support
+//@ ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL, FIXME (#58590)
```
Comment (tmandry) [inline tests/ui/test-attrs/test-panic-abort-nocapture.rs:13]:
```suggestion
//@ ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL, FIXME (#127539)
```

File: tests/ui/test-attrs/test-panic-abort.rs [hunk 2]
```diff
@@ -10,6 +10,7 @@
 //@ ignore-wasm no panic or subprocess support
 //@ ignore-emscripten no panic or subprocess support
 //@ ignore-sgx no subprocess support
+//@ ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL, FIXME (#58590)
```
Comment (tmandry) [inline tests/ui/test-attrs/test-panic-abort.rs:13]:
```suggestion
//@ ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL, FIXME (#127539)
```

**Comment by bjorn3** [comment]
This is still in the bors queue as accepted.

@bors r-

**Pull request merged**